packages/block-editor/src/components/rich-text/index.native.js

```diff
@@ -809,6 +809,7 @@ export class RichText extends Component {
 RichText.defaultProps = {
 	formattingControls: [ 'bold', 'italic', 'link', 'strikethrough' ],
 	format: 'string',
+	tagName: 'div',
```

Comment:
This is replicating the behaviour of the web where there the default tagName is a Div.

This allows the blockquote block to work properly because the <p> are surrounded by the div.

Our other blocks should work correctly because they define a tagName and don't use the default.

Reply:
> are surrounded by the div

Hmm, how do the `div`s play a role in the native mobile case, considering that this change is in the `.native.js` file? 🤔

Reply:
If you look at the file we expect a tagName in several places to be defined in order to be able to produce the HTML. It also allows the wrapping of the multiple <p> that can exist in a blockquote inside a root tag.

Reply:
Oh, so it's for proper support of the `save` method of the block?

Let's add this expanded explanation to the PR description @SergioEstevao , cool?